spec/factories/course_assessment_submissions.rb

```diff
@@ -6,8 +6,14 @@
     transient do
       grader { User.stamper }
       auto_grade true # Used only with any of the submitted or finalised traits.
+      user { creator }
```

Comment:
I prefer using the actual model attribute instead of a transient attribute for things like this.